src/batVect.mli

```diff
@@ -621,6 +628,13 @@ val find : ('a -> bool) -> 'a t -> 'a
     @raise Not_found if there is no value that satisfies [p] in the
     vect [a]. *)
 
+val find_opt : ('a -> bool) -> 'a t -> 'a option
+(** [find p a] returns [Some x] if [x] is the first element of vect [a]
```

Comment:
And in fact the phrasing of this `find_opt` is slightly different from the `find_opt` above, maybe it would be good to use the same text for both.